packages/@react-types/shared/src/collections.d.ts

```diff
@@ -128,7 +128,7 @@ export interface Collection<T> extends Iterable<T> {
   getKeys(): Iterable<Key>,
 
   /** Get an item by its key. */
-  getItem(key: Key): T,
+  getItem(key: Key | null): T,
```

Comment:
Why is null allowed here?

Reply:
`state.selectionManager.firstSelectedKey` can be null https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/actiongroup/src/ActionGroup.tsx#L396

i think you're right, i think the better thing is to include it in the if above